enhancements/ingress/route-secret-injection-for-external-certificate-management.md

```diff
@@ -0,0 +1,375 @@
+---
+title: route-secret-injection-for-external-certificate-management
+authors:
+  - "@thejasn"
+reviewers:
+  - "@Miciah"
+  - "@alebedev87"
+  - "@tgeer" 
+  - "@joelspeed"
+  - "@deads2k" 
+approvers:
+  - "@Miciah" 
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - "@joelspeed" 
+creation-date: 20222-12-13
+last-updated: 20223-01-10 
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/CFE-704
+see-also:
+  - "/enhancements/this-other-neat-thing.md"
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
```

Comment:
Do we need to keep them if they are of no use?

Reply:
Addressed.